tests/browser-asc.js

```diff
@@ -79,5 +85,23 @@ process.stdout.write(output.stdout.toString());
 console.log(">>> .stderr >>>");
 process.stdout.write(output.stderr.toString());
 console.log(">>> .text >>>");
-process.stdout.write(output.text);
-console.log(">>> .binary >>> " + output.binary.length + " bytes");
+// process.stdout.write(output.text);
+// console.log(">>> .binary >>> " + output.binary.length + " bytes");
```

Comment:
why this commented?

Reply:
I accidentally commented, because it can't run locally